index.html

```diff
@@ -256,6 +258,23 @@ <h3>The <dfn><code>VisualViewport</code></dfn> interface</h3>
                 Although it is referred to as the pinch-zoom scaling factor, it can be affected through means other than pinch-zooming. e.g. When the UA centers and zooms in a focused edit box.
               </div>
             </dd>
+            <dt><dfn>segments</dfn></dt>
+            <dd>
+              <p>
+                Returns an array of DOMRects that represent the dimensions of each existing viewport segment. 
+              </p>
+              <p>
+                A viewport segment represents the regions (and their dimensions) of the visual viewport that reside on separate displays or logical display regions. 
```

Comment:
Nit:

```
A viewport segment represents the regions
```

"A viewport segment" implies a single element of the array so I'd expect that to represent a single "region". I'd rephrase this to something like (unless I'm misunderstanding, please correct me if I'm wrong):

```
Viewport segments represent the regions...
```
or
```
A viewport segment represents a region of the visual viewport that resides on a separate display...
```

---

```
(and their dimensions)
```
Can this be removed? Doesn't a "region" imply dimensions?